kdbg/debugger.cpp

```diff
@@ -2209,6 +2232,11 @@ void KDebugger::setProgramCounter(const QString& file, int line, const DbgAddr&
     }
 }
 
+void KDebugger::handleSetDisassFlavor(const QString& s)
+{
+	emit disassFlavorChanged(s);
+}
+
```

Comment:
No, no. It is not the case that after the command was handled, the flavor was changed. It's the other way around: After the user has changed the flavor in the UI, the command is submitted in order to let Gdb know the new user preference.